content/en/agent/vector_aggregation.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Aggregating multiple agents using Vector
+kind: documentation
+further_reading:
+- link: "/logs/"
+  tag: "Documentation"
+  text: "Collect your logs"
+- link: "/agent/proxy/"
+  tag: "Documentation"
+  text: "Configure the Agent to use a proxy"
+
+---
+
+## Overview 
+
+The Datadog agent can be used along with [Vector][1], in this scenario Agents will send data
+to Vector that will aggregate data from multiple incoming Agent:
+
+`Agents ---> Vector ---> Datadog`
+
+This scenario differs from using a [proxy][2] in the extend that Vector is able to
+process data before sending them to Datadog. Vector capabilities include, among others:
+* Log sampling
+* Log deduplication to avoid duplicated log to be send
+* Transforming logs using the [Vector Remap Language][3]
+
+
+**Note**: Only logs aggregation is supported, support for other data type (metrics, processes, traces, etc.)
+will be available later.
```

Comment:
```suggestion
**Note**: Only logs aggregation is supported, support for other data type (metrics, processes, traces, etc.)
will be available at a future date.
```

Reply:
Remove note about later / future date. We refrain from adding to the docs to avoid issues.